shell/artifacts/Common/source/CopyCollection.js

```diff
@@ -0,0 +1,23 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
```

Comment:
2017 -> 2018

Reply:
argh, how do i keep missing these?! thank you.